core/src/snapshot_packager_service.rs

```diff
@@ -175,6 +176,7 @@ mod tests {
             Hash::default(),
             ArchiveFormat::TarBzip2,
             SnapshotVersion::default(),
+            SnapshotType::FullSnapshot,
```

Comment:
Add test cases for incremental snapshot.

Reply:
Sure thing! What would you like the tests to cover? Currently, all the "real" snapshot tests are either under `snapshot_utils` or `core/tests/snapshots.rs`. Are you thinking of basically a duplicate of `snapshot_packager_service::create_and_verify_snapshot()`?

Reply:
yes. Maybe we can just tag on this test case to add one more incremental snapshot?

Reply:
OK, will do.

Reply:
@lijunwangs So, looking into this test here, it's kind of funny. There is actually nothing specific about the SnapshotPackagerService being tested here. It's mostly reimplementations of the functions in snapshot_utils, then creating fake bank snapshots, building a SnapshotPackage, then passing it to functions in snapshot_utils to the actual achiving and verifying.

Since there's nothing specific to the SnapshotPackagerService being tested, and because the test itself isn't particularly "good", I'd rather not modify it here, as I don't see any added benefit. The tests in snapshot_utils and `core/tests/snapshots.rs` do already cover the test here, plus actually testing SnapshotPackagerService.

What do you think?

Reply:
Sounds good